components/button/button-mixin.js

```diff
@@ -21,6 +21,11 @@ export const ButtonMixin = superclass => class extends FocusMixin(superclass) {
 			 */
 			// eslint-disable-next-line lit/no-native-attributes
 			autofocus: { type: Boolean, reflect: true },
+			/**
+			 * Wether the controlled element is expanded. Replaces 'aria-expanded'
+			 * @type {'true' | 'false'}
+			 */
+			controlExpanded: { type: String, reflect: true, attribute: 'control-expanded' },
```

Comment:
Interesting that `aria-expanded` was already being ignored in the docs.

I'd be OK calling this just `expanded` too -- good either way.

Reply:
I think I prefer `expanded`.